manifests/base/service.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: runner-manager
+  labels:
+    app.kubernetes.io/name: runner-manager
+    app.kubernetes.io/instance: runner-manager
```

Comment:
`metadata.labels` and `spec.selector` can and should be added through base kustomization.yaml file